core/src/util/recoverable-process.ts

```diff
@@ -259,6 +261,10 @@ export class RecoverableProcess {
     return this.lastKnownPid
   }
 
+  public getProcessDescription(): string {
+    return !!this.command.description ? this.command.description : renderOsCommand(this.command)
```

Comment:
Maybe would be easier to read like this
```suggestion
    return this.command.description || renderOsCommand(this.command)
```

Reply:
Nice!